bundles/org.openhab.core.model.sitemap/src/org/openhab/core/model/sitemap/Sitemap.xtext

```diff
@@ -170,6 +171,7 @@ Number returns ecore::EBigDecimal:
 XState returns ecore::EString:
     INT | ID | STRING | FLOAT;
 
+@Override 
```

Comment:
As I am terribly fighting with the update of the xtext definition for icons, I am curious to understand why this `@override` is suddenly required. I need to learn about XText.

Reply:
It is not strictly required, but it did give a warning in the Eclipse IDE if it was not there. iD is defined by default for xtext, and his definition definitely overrides that default definition.

Reply:
Note that the grammar also gives a warning in the IDE for the Image item, as there is no required parameter (both url and item are optional), it complains about potentially not instantiating an object. So that grammar may have to be refined well.